### PR TITLE
Classification tweaks

### PR DESCRIPTION
**Pull request by Felixim0**

### CODE REVIEW

File: src/aims_ui/templates/custom_search/_macro.njk [hunk 1]
```diff
@@ -104,7 +104,7 @@
         }) }}
 
 	{% elif field.search_type == "download" %}
-	<h4>Unsure?<a href={{field.download_url}}> Click here</a>.</h4>
+	<h4>Unsure? <a href={{field.download_url}}>Click here to download a list of clasisifcations</a>.</h4>
```
Comment (steve-thorne-ONS) [inline src/aims_ui/templates/custom_search/_macro.njk]: Typo

File: src/aims_ui/models/get_fields.py [hunk 2]
```diff
@@ -164,13 +164,6 @@ def get_fields(endpoint_name):
         common_fields['england_boost'],
         common_fields['wales_boost'],
         common_fields['scotland_boost'],
-        Field(
-            'favourwelsh',
-            search_type='checkbox',
-            display_title='Flavour Welsh',
-            description=
-            'If you prefer the results to be flavoured Welsh tick this box',
```
Comment (RichardSmithONS) [inline src/aims_ui/models/get_fields.py:172]: The favourwelsh flag is a tie-breaker for when the English and Welsh versions of the address have the same number of matching parts

**Review by RichardSmithONS — COMMENTED** [review]
All looks OK but should probably change the favourwelsh description as per my comment

File: src/aims_ui/models/get_fields.py (hunk 2 above)
Comment (Felixim0) [reply src/aims_ui/models/get_fields.py:172]: I have adjusted the "boost-checkbox-description" which is the one that's now shown!
 'Boost a region. This is used as a "tie breaker" when the same address from different regions is present.'

File: src/aims_ui/templates/custom_search/_macro.njk (hunk 1 above)
Comment (Felixim0) [reply src/aims_ui/templates/custom_search/_macro.njk]: Updated to classifications

**Review by RichardSmithONS — APPROVED** [review]
LGTM

**Pull request merged**